rag/app/naive.py

```
@@ -154,17 +154,17 @@ def by_mineru(
-                # Closes #14869: when the tenant has an IMAGE2TEXT model
+                # Closes #14869: when the tenant has a VISION model
                 # configured, let the MinerU parser enrich image chunks with
                 # VLM-generated semantic descriptions (parity with deepdoc's
                 # VisionFigureParser). Best-effort — fall back silently if
                 # no vision model is available.
                 if "vision_model" not in kwargs:
                     try:
-                        vision_model_config = get_tenant_default_model_by_type(tenant_id, LLMType.IMAGE2TEXT)
+                        vision_model_config = get_tenant_default_model_by_type(tenant_id, LLMType.VISION)
                         kwargs["vision_model"] = LLMBundle(tenant_id=tenant_id, model_config=vision_model_config, lang=lang)
                     except Exception as vlm_err:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)